tools/myst/config.c

```diff
@@ -432,6 +433,32 @@ static json_result_t _json_read_callback(
                 else
                     CONFIG_RAISE(JSON_TYPE_MISMATCH);
             }
+            else if (json_match(parser, "SyslogLevel") == JSON_OK)
+            {
+                if (type == JSON_TYPE_STRING)
+                {
+                    if (strcmp(un->string, "emerg") == 0)
```

Comment:
It might be nice to have a function called something like `myst_syscall_str_to_int()` and then use this here and in the command-line args case below.

Reply:
Thanks for the suggestion! Added.